argilla/src/argilla/settings/_io/_hub.py

```diff
@@ -225,6 +223,12 @@ def _define_settings_from_features(
         questions.append(TextQuestion(name="comment", required=True))
     if not fields:
         raise SettingsError("No fields found in the dataset features. Argilla datasets require at least one field.")
+
+    # validate that atleast one field is required
+    if not any(field.required for field in fields):
```

Comment:
based on the code above, could we just check the number of fields?
```suggestion
    if len(fields) > 1:
```

Reply:
We could also just not check and always make the first field required because it's a necessity.

Reply:
Should we apply the same logic for questions? I think at least one required question is also needed.